test/models/vote_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+
+describe Vote do
+  before do
+    # Arrange
+    
+    @vote = Vote.new(user: users(:user3), work: works(:work1)) 
+  end
+
+  describe "validations" do
+    it 'is valid when all fields are present' do
+    # Act
+      result = @vote.valid?
+      # Assert
+      expect(result).must_equal true
+    end
+
+    it "don't allow to vote more than once for the same work" do
+      vote1 = Vote.create(work: works(:work1), user: users(:user1))
+      vote1.save
+      vote2 = Vote.new(work: works(:work1), user: users(:user1))
+      result = vote2.valid?
+      expect(result).must_equal false
+    end
+
+    it "needs a username(id)" do
+      vote = Vote.new(work_id: @work)
```

Comment:
This test is failing. I'm not entirely clear why, but one thing to note: you have a `@work` for this test. If you did, it should either be `work_id: @work.id` or `work: @work`